From f5bb692cee406bcc8005e107cffc53d2698d3763 Mon Sep 17 00:00:00 2001
From: Samuel Dionne-Riel <samuel@dionne-riel.com>
Date: Fri, 23 Apr 2021 01:07:25 -0400
Subject: [PATCH] rk3399-pinebook-pro: Disable cdn_dp

This may or may not help with the display on pure mainline.
---
 arch/arm/dts/rk3399-pinebook-pro-u-boot.dtsi | 13 +++++++++++++
 1 file changed, 13 insertions(+)

diff --git a/arch/arm/dts/rk3399-pinebook-pro-u-boot.dtsi b/arch/arm/dts/rk3399-pinebook-pro-u-boot.dtsi
index 1eafb40ce3..ee7232948c 100644
--- a/arch/arm/dts/rk3399-pinebook-pro-u-boot.dtsi
+++ b/arch/arm/dts/rk3399-pinebook-pro-u-boot.dtsi
@@ -41,3 +41,16 @@
 &vdd_log {
 	regulator-init-microvolt = <950000>;
 };
+
+// https://lists.infradead.org/pipermail/linux-rockchip/2020-September/022435.html
+//
+// Users can run this to test with the display port node enabled.
+//
+// ```
+// => fdt addr ${fdtcontroladdr}
+// => fdt set /dp@fec00000 status "okay"
+// ```
+
+&cdn_dp {
+	status = "disabled";
+};
-- 
2.29.2

